### PR TITLE
Smooth card elevation animation

### PR DESCRIPTION
**Pull request by halocline**
Closes https://github.com/hpe-design/design-system/issues/782

[Slack Convo](https://hpe.slack.com/archives/GPPLUK6LR/p1589234074012300)

[Preview](https://deploy-preview-776--keen-mayer-a86c8b.netlify.app/)

Adds smoother animation to cards on hover.

### CODE REVIEW

File: aries-site/src/components/cards/CardGrid.js [hunk 1]
```diff
@@ -21,7 +21,11 @@ export const CardGrid = ({ cards, ...rest }) => {
         cards.map(topic => (
           <Link key={topic.name} href={nameToPath(topic.name)} passHref>
             <ContentCard
-              as="a"
+              /* forwardedAs maintains styling provided by Higher Order 
+              Components. Styled-Components 4.3.0 docs: https://styled-components.com/docs/api#forwardedas-prop
```
Comment (ShimiSun) [inline aries-site/src/components/cards/CardGrid.js:25]: Is it still relevant on styled-component 5.0.0?

Comment (halocline) [reply aries-site/src/components/cards/CardGrid.js:25]: It is currently listed in the Styled Component docs.

File: aries-site/src/components/cards/ContentCard.js [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
   const { description, name, parent, preview } = topic;
   const [isFocused, setIsFocused] = React.useState(false);
   const darkMode = useDarkMode();
   return (
-    <Tile
+    <StyledTile
       align="start"
       background="background-front"
-      elevation={isFocused ? 'medium' : 'none'}
+      elevation={isFocused ? 'medium' : 'xsmall'}
```
Comment (taysea) [inline aries-site/src/components/cards/ContentCard.js:25]: What was the reasoning for adding default elevation? Can we leave it as flat by default?

Comment (halocline) [reply aries-site/src/components/cards/ContentCard.js:25]: Intended to communicate that the card is sitting on top of the surface. The piece of paper laying on your desk has elevation, albeit slight. When you pick it up, the elevation increases. When you set it back down, it doesn't get absorbed into surface, rather it goes back to having slight elevation.

So, attempting to mimic that experience.

It looks nice too. ;)

Comment (halocline) [reply aries-site/src/components/cards/ContentCard.js:25]: Also, FYI, working with Greg on a couple tweaks to how elevations are currently defined.

**Pull request merged**